js/bitmap-map.js

```diff
@@ -21,6 +21,7 @@ function createMap(embed)
 		"signals": "Signalling"
 	};
 
+	translations = null;
```

Comment:
Should be squashed into the commit that introduced the bug.

Reply:
Squashed.